app/src/main/res/layout/spinner_dropdown_feedback.xml

```diff
@@ -0,0 +1,11 @@
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@android:id/text1"
+    style="?android:attr/spinnerDropDownItemStyle"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
```

Comment:
Try adding android:minHeight="?android:attr/listPreferredItemHeightSmall"

The height is too small to easily click at the moment.